test/TestProjects/Inheritance/Generated/Models/BaseClassWithEnumDiscriminator.cs

```diff
@@ -12,10 +12,10 @@ namespace Inheritance.Models
     /// Please note <see cref="BaseClassWithEnumDiscriminator"/> is the base class. According to the scenario, a derived class of the base class might need to be assigned here, or this property needs to be casted to one of the possible derived classes.
     /// The available derived classes include <see cref="DerivedClassWithEnumDiscriminator"/>.
     /// </summary>
-    internal partial class BaseClassWithEnumDiscriminator
+    internal abstract partial class BaseClassWithEnumDiscriminator
```

Comment:
This might be irrelevant to the changes in this PR, but why this class is internal?

Reply:
OK - it is not used in its swagger. Actually I am expecting that this class should be removed.
I will make some follow ups on this

Reply:
@ArcturusZhang  Because `BaseClassWithEnumDiscriminator` doesn't inherit [BaseClass](https://github.com/Azure/autorest.csharp/blob/1ca6314ddf96ae913b1993c7284b6991f10e26a9/test/TestProjects/Inheritance/Inheritance.json#L55) which is used in operations. So the usage analysis of this class is none, thus making it internal.